source/crud/tests/v2/findOneAndDelete-hint.yml

```diff
@@ -0,0 +1,51 @@
+runOn:
+  - { minServerVersion: "4.4.0" }
```

Comment:
The `runOn` requirements for these tests can be updated once [SERVER-42099](https://jira.mongodb.org/browse/SERVER-42099) is merged and released.

Reply:
Updated to 4.3.1.